src/containers/FormikForm/FormikGrepCodesContent.tsx

```diff
@@ -48,6 +54,7 @@ const FormikGrepCodesContent = ({
   form,
 }: Props & tType) => {
   const [grepCodes, setGrepCodes] = useState<GrepCode[]>([]);
+  const [failedGrepCodes, setFailedGrepCodes] = useState<boolean>(false);
```

Comment:
Kunne kanskje vore en  [string] som viser fram kodene som feila?
```suggestion
  const [failedGrepCodes, setFailedGrepCodes] = useState<[string]>([]);
```